public/app/routes/routes.tsx

```diff
@@ -513,7 +517,7 @@ export function getAppRoutes(): RouteDescriptor[] {
     },
     // TODO[Router]
     // ...playlistRoutes,
-  ];
+  ].filter(isTruthy);
```

Comment:
minor question - is there an advantage to doing this over `.filter(Boolean)`? 🤔

Reply:
[isTruthy](https://github.com/grafana/grafana/blob/eea35b7e2f5072bd50584080c7b15b3609772e48/packages/grafana-data/src/types/data.ts/#L196) is a type guard, so this satisfies Typescript to have no empty values in the array.